.github/workflows/python-app.yml

```diff
@@ -16,10 +16,10 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python 3.10
+    - name: Set up Python 3.8.0
```

Comment:
why are we downgrading python version? is there some package that is update for 3.10 yet?

Reply:
this was an update on master that was merged into this branch